assets/dev/js/editor/components/template-library/views/library-layout.js

```diff
@@ -45,9 +45,15 @@ module.exports = elementorModules.common.views.modal.Layout.extend( {
 		const subscriptionPlan = subscriptionPlans[ templateAccessTier ];
 		const promotionText = elementorAppConfig.hasPro ? 'Upgrade' : `Go ${ subscriptionPlan.label }`;
 
+		const promotionLink = elementor.hooks.applyFilters(
+			'elementor/editor/template-library/template/promotion-link',
+			subscriptionPlan.promotion_url,
```

Comment:
If we add a filter here, we need to make sure the URL is still for `elementor.com`
See: `filtered-promotions-manager.php`

Reply:
@louiswol94 refactored the approach,